Woof/WoofTests/ViewModel/OwnerProfileViewModelTests/SitterMainTabViewModelTests.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+
+final class SitterMainTabViewModelTests: XCTestCase {
+    func testSitterMainTabViewModelExistsInAPI() {
+        SitterMainTabViewModel()
+    }
+
+    func testResertCurrentSitterRoleMethodExistsInAPI() {
+        SitterMainTabViewModel().resertCurrentSitterRole()
+    }
+
+    func testResertCurrentSitterRoleMethod() {
```

Comment:
The name of test does not show its intention.

Reply:
Agree. 
It was renamed on `func testResetCurrentOwnerRoleSuccessfullyChangesUserRoleToNone()`.